usaspending_api/references/management/commands/load_rosetta.py

```diff
@@ -41,36 +41,33 @@ class Command(BaseCommand):
     s3_public_url = settings.DATA_DICTIONARY_DOWNLOAD_URL
 
     def add_arguments(self, parser):
-        parser.add_argument("-p", "--path", help="filepath to a local Excel spreadsheet to load", default=None)
+        parser.add_argument("-p", "--path", help="filepath to an Excel spreadsheet to load", default=self.s3_public_url)
 
     def handle(self, *args, **options):
+        script_start_time = perf_counter()
         logger.info("Starting load_rosetta management command")
+        logger.info("Loading data from {}".format(options["path"]))
 
-        script_start_time = perf_counter()
-        rosetta_object = extract_data_from_source_file(path=options["path"])
+        try:
+            local_filepath = Path("temp_local_data_dictionary.xlsx")
```

Comment:
```suggestion
            filepath = Path("temp_local_data_dictionary.xlsx")
```
and obviously all the other places too

Reply:
Why is that?
I can see an argument to change it to `temp_filepath`. It helps to let developers know the argument could be on a remote system, but openpyxl needs a local file. It can't accept a file object

Reply:
I misread the line below. Now I see that this is the destination path, not a source value that can be overwritten by a uri